### ISSUE TITLE
Documentation doesn't match up for user creation

### ISSUE THREAD

**Issue by jtslear** [sous-chefs/postgresql#567]
### Cookbook version
7.1.1

### Chef-client version
n/a

### Platform Details
n/a

### Scenario:
Converging a recipe to create a user.

### Steps to Reproduce:
Recipe includes this:
```ruby
postgresql_user 'testuser1' do
   user 'testuser1'
   password node['password']
   action :create
   sensitive false
 end
```

### Expected Result:
* [According to documentation](https://github.com/sous-chefs/postgresql/blob/cf46a58aab5cc2c1604fbf796e075bfe8ad737e3/README.md#L330), this will create a user called `testuser1`
* [According to code](https://github.com/sous-chefs/postgresql/blob/cf46a58aab5cc2c1604fbf796e075bfe8ad737e3/resources/user.rb#L32), this uses the `testuser1` for logging in an applying the change to create `testuser1`

### Actual Result:
*  I'm going to guess that the documentation should be updated to utilize the property `create_user`

**Comment by damacus** [comment]
Yeah looks like the documentation is misleading here, `user` is the connection user, and `createuser` is the user that gets created, so the above code will try and connect to the database using the user you're about to connect with. 

I think, the `user` property should probably get rename, `connect_user` or something similar :)
